Files/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# ==================================================================================================================
+#
+# Docker to ubuntu 18.04 base image (for required libraries) used by GCBM
+#
+# Building this Docker:
+#   docker build --build-arg NUM_CPU=14 -t moja/gcbm:ubuntu-18.04 .
+#
+# ==================================================================================================================
+
+FROM moja/baseimage:ubuntu-18.04
+ENV DEBIAN_FRONTEND=noninteractive
+
+# set environment variables
+ENV ROOTDIR /usr/local/
+WORKDIR $ROOTDIR/
+
+ENV ZIPPER_VERSION 1.0.1
+ENV PATH $ROOTDIR/bin:$PATH
+ENV LD_LIBRARY_PATH $ROOTDIR/lib:$ROOTDIR/lib/x86_64-linux-gnu:$LD_LIBRARY_PATH
+ENV PYTHONPATH $ROOTDIR/lib:$PYTHONPATH
+ENV CURL_CA_BUNDLE /etc/ssl/certs/ca-certificates.crt
+ENV GDAL_DATA=/usr/local/share/gdal
+ENV GDAL_HTTP_VERSION 2
+ENV GDAL_HTTP_MERGE_CONSECUTIVE_RANGES YES
+ENV GDAL_HTTP_MULTIPLEX YES
+
+# extra deps.
+RUN apt-get update -y && apt-get install -y \
+		doxygen \
+		doxygen-latex \
+		graphviz\
+    postgis \
+	&& 	apt-get clean
+
+RUN	git clone --recursive https://github.com/sebastiandev/zipper.git \
+	&& cd zipper && git checkout e9f150516cb55d194b5e01d21a9527783e98311d && mkdir build  && cd build \
+	&& cmake .. \
+	&& make --quiet -j $NUM_CPU \
+  && make --quiet install \
+  && make clean
```

Comment:
Mirrors `Docker\Dockerfile.flint.ubuntu.18.04` from https://github.com/moja-global/FLINT. If this is the common pattern for injecting custom modules then these dependencies should be moved up into the base image.